clients/instance/ibm-pi-volume.go

```diff
@@ -286,3 +286,17 @@ func (f *IBMPIVolumeClient) BulkVolumeDelete(body *models.VolumesDelete) (*model
 	}
 	return partialResp.Payload, nil
 }
+
+// Bulk volutme attach
+func (f *IBMPIVolumeClient) BulkVolumeAttach(pvmInstanceID string, body *models.VolumesAttach) (*models.VolumesAttachmentResponse, error) {
```

Comment:
BulkVolumeDetach named the first parameter `pvmID`. Let's be consistent.

Reply:
Done